sepal_ui/sepalwidgets/app.py

```diff
@@ -108,6 +126,46 @@ def __init__(self, title, icon=None, card=None, href=None, **kwargs):
         # call the constructor
         super().__init__(**kwargs)
 
+        # cannot be set as a class member because it will be shared with all
+        # the other draweritems.
+        self.alert_badge = v.ListItemAction(
+            children=[v.Icon(children=["fas fa-circle"], x_small=True, color="red")]
+        )
+
+        if model:
```

Comment:
Could it be squeezed to `if model and not bind_var` ?

Reply:
The `if` statement is not only used to trigger the exception, if the model is set, we link the `traits`, so we don't need to use the `else`.